.github/workflows/tests_and_reports.yml

```diff
@@ -126,11 +127,51 @@ jobs:
           path: bundle-visualizer
           retention-days: 1
 
+  lighthouse:
+    name: "Lighthouse"
+    runs-on: "ubuntu-latest"
+    timeout-minutes: 60
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Chrome
+        uses: browser-actions/setup-chrome@latest
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+          cache: npm
+
+      - name: Install
+        run: |
+          npm ci
+          npm i -g lighthouse lighthouse-batch
+
+      - name: Generate Lighthouse reports
+        run: |
+          npm run build
+          npm run preview &
+          sleep 5
+          lighthouse-batch --use-global --html --out lighthouse-reports/mobile --file .lighthouse/urls.txt --params "--only-categories=\"performance,accessibility,best-practices,seo\""
+          lighthouse-batch --use-global --html --out lighthouse-reports/desktop --file .lighthouse/urls.txt --params "--only-categories=\"performance,accessibility,best-practices,seo\" --preset=desktop"
```

Comment:
I tried installing these as dev dependencies and writing these commands under scripts in `package.json`, but I was running into all sorts of issues both locally and in CI, so I opted for the first thing that worked, which is what we've got here (global installs and commands written out in the workflow).

Reply:
That's fine by me, it's working!